react-client/src/App.tsx

```diff
@@ -14,201 +14,32 @@
+import { ColorSchemeScript, DirectionProvider, MantineProvider } from '@mantine/core';
 import { Notifications } from '@mantine/notifications';
+import '@mantine/core/styles.css';
+import '@mantine/dropzone/styles.css';
+import '@mantine/notifications/styles.css';
 
-import { useEffect } from 'react';
-import {
-  BrowserRouter as Router,
-  Route,
-  Routes,
-  Navigate,
-} from 'react-router-dom';
-import rtlPlugin from 'stylis-plugin-rtl';
-import { MoonStars, Sun } from 'tabler-icons-react';
-
-import './services/http-interceptor';
-import About from './components/About/About'
-import Accounts from './components/Accounts/Accounts'
-import Actions from './components/Actions/Actions';
-import Home from './components/Home/Home';
-import LanguagesMenu from './components/LanguagesMenu/LanguagesMenu';
-import Login from './components/Login/Login'
-import Logs from './components/Logs/Logs'
-import Player from './components/Player/Player';
-import PlayerLogin from './components/PlayerLogin/PlayerLogin';
-import Settings from './components/Settings/Settings';
-import SharedContent from './components/SharedContent/SharedContent';
-import UserMenu from './components/UserMenu/UserMenu';
-import NavbarContext from './contexts/navbar-context';
-import SessionContext from './contexts/session-context';
-import { I18nProvider } from './providers/i18n-provider';
-import { AccountsProvider } from './providers/accounts-provider';
-import { PlayerEventProvider } from './providers/player-server-event-provider';
-import { ServerEventProvider } from './providers/server-event-provider';
-import { SessionProvider } from './providers/session-provider';
-import { refreshAuthTokenNearExpiry, setAxiosAuthorization } from './services/auth-service';
-import { NavbarProvider } from './providers/navbar-provider';
-
-type ExtendedCustomColors = 'darkTransparent' | 'lightTransparent' | DefaultMantineColor;
-
-declare module '@mantine/core' {
-  export interface MantineThemeColorsOverride {
-    colors: Record<ExtendedCustomColors, Tuple<string, 10>>;
-  }
-}
+import UmsApp from './UmsApp';
 
 function App() {
-  const [rtl, setRtl] = useLocalStorage<boolean>({
-    key: 'mantine-rtl',
-    defaultValue: false,
-  });
-  const [colorScheme, setColorScheme] = useLocalStorage<ColorScheme>({
-    key: 'mantine-color-scheme',
-    defaultValue: 'dark',
-    getInitialValueInEffect: true,
-  });
-  const toggleColorScheme = (value?: ColorScheme) => {
-    setColorScheme(value || (colorScheme === 'dark' ? 'light' : 'dark'));
-  }
-
-  const rtlCache = createEmotionCache({
-    key: 'mantine-rtl',
-    stylisPlugins: [rtlPlugin],
-  });
-  setAxiosAuthorization();
-  useEffect(() => {
-    refreshAuthTokenNearExpiry();
-  });
 
   return (
-    <ColorSchemeProvider colorScheme={colorScheme} toggleColorScheme={toggleColorScheme}>
+    <DirectionProvider>
+      <ColorSchemeScript defaultColorScheme='dark' localStorageKey='mantine-color-scheme' />
```

Comment:
It might be cool to default to the system preference as described at https://mantine.dev/changelog/7-0-0/#system-color-scheme-support
What do you think?